rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -290,44 +287,41 @@ type Payout: <"Represents the set of future cashflow methodologies in the form o
-            and forwardPayout -> priceQuantity -> reset is absent
+            and settlementPayout -> priceQuantity -> reset is absent
             and optionPayout -> priceQuantity -> reset is absent
             and cashflow -> priceQuantity -> reset is absent
 
     condition NotionalResetInterestRatePayoutExists: <"As the performancePayout->payoutQuantity->reset attribute applies to return swaps, the interestRatePayout needs to be present alongside it.">
         if performancePayout -> priceQuantity -> reset contains True
         then interestRatePayout exists
 
-type SettlementPayout extends PayoutBase:  <"This is a payout data type used to define the commitment to settle a trade in an Asset or TransferableProduct. Examples: a purchase of a cash security; an FX trade, either spot or forward.">
-    underlier Underlier (1..1)  <"A transferable Asset with or without the addition of specific EconomicTerms.">
+choice Underlier:  <"The underlying financial product that will be physically or cash settled, which can be of any type, eg an asset such as cash or a security, a product, or the cash settlement of an index rate.  Conditions are usually applied when used in a data type, such as a payout, to ensure this aligns with the use case.">
+    Observable  <"Specifies the object to be observed for a price, it could be an asset or a reference.">
+        [metadata address "pointsTo"=PriceQuantity->observable]   
+    Product  <"Enables either a TransferableProduct or a NonTransferableProduct to be used in an underlier.">
 
-choice Underlier:  <"The underlier of a payout which can be either an Asset or a Transferable Product, which is an Asset with Economic Terms.">
-    Asset  <"Identifies a transferable asset.">
-    TransferableProduct  <"Defines the transferable asset with an identifier and economic terms.">
+choice Product:  <"Enables either a TransferableProduct or a NonTransferableProduct to be used in an underlier.">
+    TransferableProduct    <"A TransferableProduct is a type of financial product which can be held or transferred, represented as an Asset with the addition of specific EconomicTerms.">
+    NonTransferableProduct <"The non-transferable product data type represents a product that can be traded (as part of a TradableProduct) but cannot be transferred to others.">
 
-type Product: <"Defines the product that is the subject of a tradable product definition, an underlying product definition, a physical exercise, a position, or other purposes.">
+type NonTransferableProduct:  <"A data type to specify the financial product's economic terms, alongside the product identification and product taxonomy. The non-transferable product data type represents a product that can be traded (as part of a TradableProduct) but cannot be transferred to others.  It is meant to be used across the pre-execution, execution and (as part of the Contract) post-execution lifecycle contexts.">
```

Comment:
See related comment.

If `NonTransferableProduct` doesn't extend `ProductBase`, delete the latter.

Reply:
Leaving for the moment as I think this is how ETD's lost their taxonomy

Reply:
See above comment - Remove `ProductBase`

Reply:
done